security/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion project.ext.targetSdkVersion
+
+    defaultConfig {
+        minSdkVersion project.ext.minSdkVersion
+        targetSdkVersion project.ext.targetSdkVersion
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+            returnDefaultValues = true
+        }
+    }
+}
+
+dependencies {
+    implementation project(path: ":core")
+
+    implementation "com.scottyab:rootbeer-lib:${project.ext.rootbeer_version}"
+
+    testImplementation "junit:junit:${project.ext.junit_version}"
+    testImplementation "org.mockito:mockito-core:${project.ext.mockito_version}"
+    testImplementation "org.robolectric:robolectric:${project.ext.robolectric_version}"
+}
```

Comment:
We need to add `apply from: "../gradle-mvn-push.gradle"` to this file for the security module to be picked up by gradle

Reply:
Yeah, I forgot to commit them! Good catch!

Reply:
It is @secondsun catch not mine haha